webpack.common.js

```diff
@@ -26,6 +28,7 @@ module.exports = {
 			{
 				test: /\.(css|scss)$/,
 				use: [
+					'style-loader',
```

Comment:
Willen we de CSS geforceerd in de JS hebben, of mag/moet dit afgesplitst worden in een aparte file? Wat vind jij, @FricoRico ?